google/cloud/forseti/common/gcp_type/iam_policy.py

```diff
@@ -187,7 +191,27 @@ def create_from(cls, binding):
         """
         if isinstance(binding, type(cls)):
             return binding
-        return cls(binding.get('role'), binding.get('members'))
+        try:
+            return cls(binding.get('role'), binding.get('members'))
+        except errors.InvalidIamPolicyMemberError:
+            LOGGER.warn(
+                'Invalid IAM policy member: %s.', binding.get('members'))
+            return None
+
+    def merge(self, other):
```

Comment:
Hmm .. there's nothing else to merge but the members :)

Reply:
Then why do we need this comment to clarify where this is used?  That's why I suggested you can make the code self-documenting and drop the comment.  Otherwise, we could be merging roles (again at the point where this is used).
```
bucket_binding.merge_members(ancestor_binding)
```
![image](https://user-images.githubusercontent.com/6977544/36927761-ae43499c-1e34-11e8-9202-ab7f16ff0d58.png)